R/utils.R

```diff
@@ -358,3 +358,10 @@ cli_with_whiteline_escapes <- function(x, fn) {
   x <- gsub("__NEW_LINE__", "\n", x, fixed = TRUE)
   x
 }
+
+style_rlang_run <- function(code) {
+  style_hyperlink(
+    paste0("`rlang::", code, "`"),
+    paste0("rstudio:run:rlang::", code)
```

Comment:
We were discussing with @romainfrancois about making the protocol more general:

- Check if `R_CLI_HAS_HYPERLINK_RUN` envvar is set.
- If set, hyperlink to `ide:run:`